### PR TITLE
v6.4.0

### PR DESCRIPTION
**Pull request by adamraine**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,119 @@
+Thanks to our new contributors 👽🐷🐰🐯🐻! 
+Denis Seleznev @hcodes
+Irfan Maulana @mazipan
+James Garbutt @43081j
+Matt Hobbs @Nooshu
+Mohammed J. Razem @moerazem
+Pujitha.E @pujitha7
+Quentin @qwright10
+Adriana Ixba @adrianaixba
```
[the remark this answers is not in the document]
Comment (adamraine) [reply changelog.md:9]: Looks like adriana was the only one

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,119 @@
...
+
+* large-javascript-libraries: move to experimental ([#11484](https://github.com/GoogleChrome/lighthouse/pull/11484))
+
+## New Audits
+
+* unsized-images reland ([#11340](https://github.com/GoogleChrome/lighthouse/pull/11340))
+
+## Core
+
+* autocomplete: add chrome suggestions, invalid warning ([#11342](https://github.com/GoogleChrome/lighthouse/pull/11342))
```
[the remark this answers is not in the document]
Comment (adamraine) [reply changelog.md]: I moved some around. Let me know if I missed some important ones.

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,122 @@
...
+
+* unsized-images reland ([#11340](https://github.com/GoogleChrome/lighthouse/pull/11340))
+
+## Core
+
+* tracehouse: add CPU trace profiler model ([#11072](https://github.com/GoogleChrome/lighthouse/pull/11072))
+* stack-packs: add Drupal pack ([#10522](https://github.com/GoogleChrome/lighthouse/pull/10522))
+* stack-packs: move to lighthouse-stack-packs npm package ([#11370](https://github.com/GoogleChrome/lighthouse/pull/11370))
+* axe: use our html snippet rather than axe's ([#11362](https://github.com/GoogleChrome/lighthouse/pull/11362))
+* driver: don't clear indexedb, websql, or localstorage before run ([#11438](https://github.com/GoogleChrome/lighthouse/pull/11438))
```
Comment (patrickhulce) [inline changelog.md:34]: this feels somewhat notable as well, it could potentially be significant to a subset of pages

File: changelog.md [hunk 4]
```diff
@@ -1,3 +1,122 @@
...
+Thanks to our new contributors 👽🐷🐰🐯🐻! 
+
+Denis Seleznev @hcodes
+Irfan Maulana @mazipan
+James Garbutt @43081j
+Matt Hobbs @Nooshu
+Mohammed J. Razem @moerazem
+Pujitha.E @pujitha7
+Quentin @qwright10
+andreizet @andreizet
```
Comment (patrickhulce) [inline changelog.md:18]:
```suggestion
- Denis Seleznev @hcodes
- Irfan Maulana @mazipan
- James Garbutt @43081j
- Matt Hobbs @Nooshu
- Mohammed J. Razem @moerazem
- Pujitha.E @pujitha7
- Quentin @qwright10
- andreizet @andreizet
```

**Review by patrickhulce — APPROVED** [review]
LGTM 🎉

**Pull request merged**